addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -92,9 +92,19 @@ export class PropertiesField extends Component {
      * @returns {array}
      */
     get propertiesList() {
-        const propertiesValues = JSON.parse(
-            JSON.stringify(this.props.record.data[this.props.name] || [])
-        );
+        const propertiesValues = (this.props.record.data[this.props.name] || []).map(property => {
+            let value;
+            if (["date", "datetime"].includes(property.type)) {
+                // Date and Datetime are immutable luxon objects.
+                value = property.value;
```

Comment:
can't we return here already ?
and if the if statement is not met, return directly the deep copy?

Reply:
Hello @dbeguin ,

I have made the changes.
Thanks for the review.